### PR TITLE
ARXIVCE-742: 3 locations where we could put a message

### PR DESCRIPTION
**Pull request by bmaltzan**
for the user about being in gcp.

### CODE REVIEW

**Review by bdc34 — APPROVED** [review]

File: browse/templates/base.html [hunk 1]
```diff
@@ -34,12 +34,13 @@
         <div class="column" id="cu-logo">
           <a href="https://www.cornell.edu/"><img src="{{ url_for('static', filename='images/icons/cu/cornell-reduced-white-SMALL.svg') }}" alt="Cornell University" /></a>
         </div>
-        {%- set rd_int = request_datetime.strftime("%Y%m%d%H%M")|int -%}
-        {%- if rd_int >= 202305010100 and rd_int <= 202307170100 -%}
+        {%- if config["BROWSE_MINIMAL_BANNER_ENABLED"] and rd_int >= 202310010100 and rd_int <= 202312300100 -%}
         <div class="column banner-minimal">
-            <a href="https://info.arxiv.org/hiring/">We are hiring</a></p>
+            <a href="https://info.arxiv.org/about/index.html">To the cloud!</a></p>
```
Comment (SBBCornell) [inline browse/templates/base.html:39]: @alisonhofer might have a different take, but arXiv's tone is usually more matter of fact. What about "Moving to the cloud" or "Server stability update"?

File: browse/templates/home/special-message.html [hunk 2]
```diff
@@ -19,3 +21,15 @@ <h2>TeX Live 2023 Upgrade</h2>
   <p><a href="https://info.arxiv.org/hiring/">View open roles</a></p>
 </div>
 {%- endif -%}
+
+{%- if config['BROWSE_SPECIAL_MESSAGE_ENABLED'] and rd_int >= 202310010100 and rd_int <= 202312300100 -%}
+<div class="message-special dark column">
+  <span class="label">arXiv is moving!</span>
+  <p>Experience the future with our cloud-powered app today!</p>
```
Comment (SBBCornell) [inline browse/templates/home/special-message.html:28]:
```suggestion
  <p>arXiv is preparing for our next 30 years by transitioning from our current aging infrastructure to stable cloud-hosted services. </p>
```

File: browse/templates/home/special-message.html [hunk 3]
```diff
@@ -19,3 +21,15 @@ <h2>TeX Live 2023 Upgrade</h2>
   <p><a href="https://info.arxiv.org/hiring/">View open roles</a></p>
 </div>
 {%- endif -%}
+
+{%- if config['BROWSE_SPECIAL_MESSAGE_ENABLED'] and rd_int >= 202310010100 and rd_int <= 202312300100 -%}
+<div class="message-special dark column">
+  <span class="label">arXiv is moving!</span>
```
Comment (SBBCornell) [inline browse/templates/home/special-message.html:27]:
```suggestion
  <span class="label">Cloud transition</span>
```

File: browse/templates/user_banner.html [hunk 4]
```diff
@@ -13,28 +13,31 @@
...
-{%- if BANNER_START <= request_datetime.strftime('%Y%m%d') and BANNER_END > request_datetime.strftime('%Y%m%d') %}
+{%- if config['BROWSE_USER_BANNER_ENABLED'] and (BANNER_START <= request_datetime.strftime('%Y%m%d')) and (BANNER_END > request_datetime.strftime('%Y%m%d')) %}
 <aside class="slider-wrapper bps-banner dark" >
   <div class="copy-donation bps-banner">
-    <h1>Example banner</h1>
-    <p>Banner text would go here.</p>
-    <p><a href="https://arxiv.org/support/bug_report">Report bugs here</a></p>
+    <h1>Cloud Transition</h1>
+    <p>We're making your experience even smoother by migrating to the cloud.</p>
+    <p><a href="https://info.arxiv.org/about/index.html">Report bugs here</a></p>
```
Comment (SBBCornell) [inline browse/templates/user_banner.html:32]:
```suggestion
    <p><a href="https://arxiv-org.atlassian.net/servicedesk/customer/portal/1l">Report an issue</a></p>
```

File: browse/templates/user_banner.html [hunk 5]
```diff
@@ -13,28 +13,31 @@
...
 {%- macro content(request_datetime) -%}
-{%- if BANNER_START <= request_datetime.strftime('%Y%m%d') and BANNER_END > request_datetime.strftime('%Y%m%d') %}
+{%- if config['BROWSE_USER_BANNER_ENABLED'] and (BANNER_START <= request_datetime.strftime('%Y%m%d')) and (BANNER_END > request_datetime.strftime('%Y%m%d')) %}
 <aside class="slider-wrapper bps-banner dark" >
   <div class="copy-donation bps-banner">
-    <h1>Example banner</h1>
-    <p>Banner text would go here.</p>
-    <p><a href="https://arxiv.org/support/bug_report">Report bugs here</a></p>
+    <h1>Cloud Transition</h1>
+    <p>We're making your experience even smoother by migrating to the cloud.</p>
```
Comment (SBBCornell) [inline browse/templates/user_banner.html:31]:
```suggestion
    <p>We're making arXiv even more stable by transitioning to the cloud.</p>
```

**Review by SBBCornell — COMMENTED** [review]
Not sure if you were looking for word-smithing, but I made some suggestions. 

One other comment is that having both the main banner and the minimal banner seems like a lot of messaging. The minimal banner is great if we can get the message across in <5 words. But if not, the main banner gives us room for a whole sentence or two!

File: browse/templates/home/special-message.html (hunk 2 above)
Comment (bdc34) [reply browse/templates/home/special-message.html:28]: I'd suggest "arxiv is preparing for our next 30 years by transitioning to cloud-hosted services"

Comment (cbf66) [reply browse/templates/home/special-message.html:28]: Let's not put this out there until we have a landing page that explains the plan more fully. I don't want to introduce the whole idea of the cloud transition with a one sentence message.

**Review by cbf66 — COMMENTED** [review]
Please hold this until I have a landing page ready. Give me an hour.

**Comment by bmaltzan** [comment]
> Not sure if you were looking for word-smithing, but I made some suggestions.

Definitely.
Mine was just some random placeholder text.

**Comment by bdc34** [comment]
I changed the base to `develop` the gcp-xyz branches are not in use any more. I'll delete them soon.

**Comment by cbf66** [comment]
> Please hold this until I have a landing page ready. Give me an hour.

Done [https://info.arxiv.org/help/cloud.html](url)

**Pull request merged**